engine/client/voice.c

```diff
@@ -202,7 +203,7 @@ static uint Voice_GetOpusCompressedData( byte *out, uint maxsize, uint *frames )
 		if( !voice.input_file )
 		{
 			// adjust gain before encoding, but only for input from voice
-			Voice_ApplyGainAdjust((opus_int16*)(voice.input_buffer + ofs), voice.frame_size);
+			Voice_ApplyGainAdjust((opus_int16*)(voice.input_buffer + ofs), voice.frame_size, voice_transmit_scale.value);
```

Comment:
`voice_scale` is probably not documented properly.

It looks like it's an outgoing volume scale, rather than incoming, as it's value being passed to autogain.

Reply:
`voice_scale` used in rawchan volume, therefore it influences others voice sound volume. `voice_transmit_scale` used in autogain, which is used in process of sending voice data from client to others.

Reply:
I suggest keeping compatibility with GoldSrc here.

voice_scale is for outgoing data.
voice_volume (maybe?) is for incoming data.

Reply:
Of course, let's keep it if it's possible. I'll check how it was done in GoldSrc.

Reply:
`voice_scale` already works as in GoldSrc. Also, there is no any functional analog for `voice_transmit_scale` in GoldSrc.